lib/base-config.js

```diff
@@ -28,4 +28,8 @@ module.exports = () => ({
     snapshotSerializers: [
         require.resolve('jest-serializer-path'),
     ],
+    testMatch: [
+        '**/__tests__/**/*.[jt]s?(x)',
+        '**/?(*.)test.[jt]s?(x)',
+    ],
```

Comment:
@satazor I'm now wondering what's the point of having the matching pattern `'**/__tests__/**/*.[jt]s?(x)'`. Doesn't `'**/?(*.)test.[jt]s?(x)'` pretty much cover all the scenarios we're looking for?

Reply:
If I have co-localized utils to support my tests in the `__tests__` folder, Jest will still detect them.

Reply:
That pattern allow us to have tests files with the suffix in test folders. That’s pattern is Jest defaults.

Reply:
Fixed my comment.

Reply:
If we want to enforce all test files to end with `.test.js`, even if they are already in `__tests__` or `test`, then we may tweak the pattern.

I don’t have a strong opinion on this one, however maybe we should keep the pattern close to Jest defaults? (Disallowing just spec ones)

Reply:
> If I have co-localized utils to support my tests in the `__tests__` folder, Jest will still detect them. We don't want this, right?

That’s a good point to actually restrict test files to `.test.js`.

Reply:
I've just dropped `'**/__tests__/**/*.[jt]s?(x)'` pattern.

Reply:
@moxystudio/developers, other opinions are welcomed. 💪

Reply:
> I've just dropped '**/__tests__/**/*.[jt]s?(x)' pattern.

agree 👍